packages/office-ui-fabric-react/src/components/Persona/Persona.test.tsx

```diff
@@ -4,12 +4,15 @@ import { setRTL } from '../../Utilities';
 import { Persona } from './Persona';
 import { PersonaInitialsColor } from './Persona.Props';
 import { mount, ReactWrapper } from 'enzyme';
-import * as stylesImport from './Persona.scss';
-const styles: any = stylesImport;
 
 const testImage1x1 = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAAC0lEQVQImWP4DwQACfsD/eNV8pwAAAAASUVORK5CYII=';
+const STYLES = {
+  darkRed: '.ms-Persona-initials--darkRed',
+  initials: '.ms-Persona-initials',
+  lightBlue: '.ms-Persona-initials--lightBlue',
+  red: '.ms-Persona-initials--red',
```

Comment:
I hard code these values because other styles were not being processed. Because sass is not being processed?